TakingITGlobal/commit2act
# [TIG 795] - Implemented i18n in TextInput components

# Description
[TIG-795 - Clickup Task](https://app.clickup.com/t/86dt3b5jr)

In this PR, I implemented the translations for the TextInput components. Everything should look the same as the PR  #120, just the helperText and aria-label should now use il8n strings. 

The criteria for this PR:

- [ ] `helperText` uses il8n strings and it shows up on `Outlined Email` , `Outlined Password Required`  `Outlined Required` on Storybook when entering an invalid email or an empty field
- [ ] `aria-label` uses il8n strings

If you are viewing this PR, please also check out PR  #120. 

# Testing
 **Testing is optional, since it is the same as PR #120** but always feel free to run it
1. Pull down and install the branch (`npm install`)
2. Run all tests (`npm run test:basic`) and confirm they pass
3. Run Storybook (`npm run storybook`)

# Images  
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/2d5ae3d5-4e48-414b-ace9-075a7af26efc)
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/a1f26b89-3bea-4421-9361-c3402ea40f86)
